src/lib/menu/menu-directive.ts

```diff
@@ -76,8 +78,11 @@ export class MdMenu implements AfterContentInit, MdMenuPanel, OnDestroy {
   /** Subscription to tab events on the menu panel */
   private _tabSubscription: Subscription;
 
+  /** Stream that emits whenever the component is intialized. */
+  private _initialized = new Subject<void>();
```

Comment:
Instead of creating another Subject to track initialization, how about using `NgZone.onStable` ?

Reply:
I’m not sure whether the timing is the same.

Reply:
I stand corrected, it does work.